src/scales/scale.ts

```diff
@@ -1,6 +1,11 @@
 ///<reference path="../reference.ts" />
 
 module Plottable {
+
+  export interface ScaleCallback<S extends Scale<any, any>> {
+    (scale: S): any;
```

Comment:
Noticed that none of the callbacks actually utilize the scale except for in test code.  We should probably not have the scale be an enforced argument?